social-network/src/main/java/com/levi9/socialnetwork/Service/impl/UserServiceImpl.java

```diff
@@ -59,7 +59,17 @@ public ResponseEntity<User> getUserById(Long userId)
 		User user = userRepository.findById(userId).orElseThrow(() -> new ResourceNotFoundException("User not found for this id :: " + userId));
 	 return ResponseEntity.ok().body(user);
 	}
-	
+
+	@Override
+	public User findUserById(Long userId) throws ResourceNotFoundException {
+		return userRepository.findById(userId).orElseThrow(() -> new ResourceNotFoundException("User not found for this id :: " + userId));
```

Comment:
Extract the message in a constant and use it here and in getUserById(Long userId).